server/src/main/java/org/apache/druid/server/coordinator/rules/PeriodLoadRule.java

```diff
@@ -74,13 +72,6 @@ public boolean isIncludeFuture()
     return includeFuture;
   }
 
-  @Override
-  @JsonProperty
-  public Map<String, Integer> getTieredReplicants()
-  {
-    return tieredReplicants;
-  }
-
   @Override
   public int getNumReplicants(String tier)
```

Comment:
This method is also common across the 3 load rule implementations, could be moved into `LoadRule`.